### PR TITLE
Add SAML Bridge Configuration Wizard Console Application to samlbridge solution.

### PR DESCRIPTION
**Pull request by lavanyachandramouli**
This is the stand alone application related to saml bridge configuration. It helps end users to configure artifact consumer url.

### CODE REVIEW

File: SAMLBridgeConfiguation/SAMLBridgeConfiguation.csproj [hunk 1]
```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (wiarlawd) [inline SAMLBridgeConfiguation/SAMLBridgeConfiguation.csproj]: Do these 4.0 references scattered around refer to the .NET Framework version, and do they mean that this tool (and possibly also SAML Bridge) requires version 4.0 to run? In some places here, the old values from GSARKS were 2.0 or 3.5.

**Comment by lavanyachandramouli** [comment]
Created Config-Tool as a top level folder. Corrected typo on SAMLBridgeConfiguration project.
Supported .net Framework will be 2.0 for all the solution files.
Please review.

**Comment by wiarlawd** [comment]
So you added Config-Tool as a new directory because the tools require the subdirectory to be named "SAMLBridgeConfiguration" (or rather, to match the project name)? I don't know whether it makes sense to keep a separate directory layer just for better naming. If we do keep it, please use lowercase "config-tool" (as requested, and matching the existing subdirectories).

Why does this need a separate directory layer, where gsa-simulator and saml-bridge do not? What would need to be changed (e.g., in the installed configuration tool) if we just have a single config-tool directory with the .cs files and the rest directly in that?

Why do we have the bin files and such in the repository?

John L

**Comment by lavanyachandramouli** [comment]
I have conflicts while pushing changes from my local branch. Creating another pull request.

**Pull request closed without merging**